packages/cli/odahuflow/cli/parsers/local/packaging.py

```diff
@@ -60,6 +62,136 @@ def cleanup_containers():
     cleanup_packaging_docker_containers()
 
 
+def fetch_local_entities(manifest_file, manifest_dir):
+    """
+    Collect entities from manifest files in local FS and return a result
+    Manifests can be collected from file or files inside a directory
+    Manifests from different sources are combined together
+    :param manifest_file: manifest file
+    :param manifest_dir: directory with manifest files
+    :return:
```

Comment:
```suggestion
```
These are redundant, in my opinion. I'd either remove them at all or extend the description.